src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__item-img a {
+    display: block;
+    margin-bottom: 40px;
+  }
+
+  &__item-img img {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__item-name {
+    color: $title-text-color;
+    margin-bottom: 4px;
+
+    @include title-text-font;
+  }
+
+  &__item-code {
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__item-review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review a {
+    text-decoration: none;
+    color: $main-text-color;
+  }
+
+  &__item-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__item-price p:first-child {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__item-price p:last-child {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__btn-buy {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__btn-buy button {
```

Comment:
same about the combined selector (but here you need only `a` tag)
Check if the button has correct styles, looks bad in the Demo link:

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/e73f9268-826f-44c9-a1df-226578343cd0)